search/update_attribute_configuration.py

```diff
@@ -71,7 +70,8 @@ def update_product(product_to_update_id):
     print(updated_product)
 
     print('---Wait 5 minutes to be sure the catalog has been indexed after the changes---:')
-    time.sleep(300)
+    #time.sleep(300)
```

Comment:
I think sleep here should not be commented out (at least basing on the print above and below)